test/conformance/exp_command_buffer/update/local_memory_update.cpp

```diff
@@ -8,15 +8,19 @@
 #include <cstring>
 
 // Test that updating a command-buffer with a single kernel command
-// taking a local memory argument works correctly.
-
+// taking local memory arguments works correctly.
 struct LocalMemoryUpdateTestBase
     : uur::command_buffer::urUpdatableCommandBufferExpExecutionTest {
     virtual void SetUp() override {
         program_name = "saxpy_usm_local_mem";
         UUR_RETURN_ON_FATAL_FAILURE(
             urUpdatableCommandBufferExpExecutionTest::SetUp());
 
+        if (backend == UR_PLATFORM_BACKEND_LEVEL_ZERO) {
+            GTEST_SKIP()
+                << "Local memory argument update not supported on Level Zero.";
+        }
+
         // HIP has extra args for local memory so we define an offset for arg indices here for updating
```

Comment:
apologies but as long as we're in here might as well fix the formatting
```suggestion
        // HIP has extra args for local memory so we define an offset for arg
        // indices here for updating
```